### PR TITLE
Orb Art Rework, some fixes

### PR DESCRIPTION
**Pull request by ScipioWright**
Reworking orb art to use child entities to reduce the number of unique assets required, as well as the file size for the mod. Also makes it much easier to update art assets.

This is a draft because I didn't finish yet, and there's a placeholder for trap items (using an X instead of the unique art).

Also included a fix for the load key thing not working -- Heinermann said he'd look into why the other method didn't work. I left it commented out with a todo next to it.

### CODE REVIEW

**Comment by heinermann** [comment]
You can probably include multiple SpriteComponents in the XML file for the graphic itself too.

**Comment by ScipioWright** [comment]
Changes made, SpriteComponents componented, and ready to merge.

**Comment by heinermann** [comment]
Sorry just saw, will get to it tomorrow.

File: data/archipelago/scripts/item_utils.lua [hunk 1]
```diff
@@ -39,7 +39,7 @@ function SpawnItem(item_id, traps)
 		Log.Info("Perk spawned")
 	elseif #item.items > 0 then
 		EntityLoadAtPlayer(item.items[Random(1, #item.items)])
-		Log.Info("Item spawned")
+		Log.Info("Item spawned" .. item.items[Random(1, #item.items)])
```
Comment (heinermann) [inline data/archipelago/scripts/item_utils.lua:42]: This is going to choose a different random item to log, you should pull it out into a `local item_to_spawn = item.items[Random(1, #item.items)]` above `EntityLoadAtPlayer`.

File: init.lua [hunk 2]
```diff
@@ -232,15 +234,27 @@ function RECV_MSG.Connected(msg)
 
 	Globals.Seed:set(slot_options.seed)
 
-	if Globals.LoadKey:get() ~= "1" then
```
Comment (heinermann) [inline init.lua:235]: What was wrong with `Globals.LoadKey`?

Comment (heinermann) [reply init.lua:235]: I guess I'll look into it

**Pull request merged**